src/Copy/BrandCopy.elm

```diff
@@ -50,170 +50,170 @@ brandCopy : Key -> Copy
 brandCopy key =
     case key of
         SiteTitle ->
-            CopyText "Contemplating Action"
+            CopyText "✨"
 
         AppTitle ->
-            CopyText "Contemplating Action"
+            CopyText "✨"
 
         CallToActionOne ->
             CallToAction
                 { action = DesktopInteractive
-                , category = "survey"
-                , icon = "check-square-o"
-                , href = "https://Contemplating-action.typeform.com/to/zUOTYA"
+                , category = "✨"
+                , icon = "✨"
+                , href = "✨"
                 , displayHref = ""
-                , promptLong = "Take part in our survey"
-                , promptShort = "Survey"
+                , promptLong = "✨"
+                , promptShort = "✨"
                 }
 
         CallToActionTwo ->
             CallToAction
                 { action = DesktopInteractive
                 , category = "page"
-                , icon = "question-circle-o"
-                , href = "/stories"
-                , displayHref = "Find Out More"
-                , promptLong = "Find Out More"
-                , promptShort = "Find Out More"
+                , icon = "✨"
+                , href = "#"
```

Comment:
a # left in but I think OK oto merge since this will not be in the release copy.